apt_offline_core/AptOfflineCoreLib.py

```diff
@@ -2072,7 +2086,7 @@ def main():
         parser_set = subparsers.add_parser('set', parents=[global_options])
         parser_set.set_defaults(func=setter)
         
-        parser_set.add_argument('set',
+        parser_set.add_argument('set', nargs='?',
```

Comment:
Okay. So as I understand, this is proposed as a combination for --basepath. So this may not be needed either, if we want the args to be explicitly mentioned.